mipengine/controller/api/error_handlers.py

```diff
@@ -9,15 +9,23 @@
 from mipengine.controller.algorithm_executor import AlgorithmExecutionException
 from mipengine.controller import controller_logger as ctrl_logger
 
+logger = ctrl_logger.getLogger(__name__)
+
 error_handlers = Blueprint("error_handlers", __name__)
 
-INSUFFICIENT_DATA_ERROR_MESSAGE = "The algorithm could not run with the input provided because there are insufficient data."
+INSUFFICIENT_DATA_ERROR_MESSAGE = (
+    "The algorithm could not run with the input "
+    "provided because there are insufficient data."
+)
+
+ALGORITHM_EXUCUTION_ERROR = "An error occured during the execution of the algorithm"
 
 
 class HTTPStatusCode(enum.IntEnum):
     BAD_REQUEST = 400
     BAD_USER_INPUT = 460
     INSUFFICIENT_DATA_ERROR = 461
+    ALGORITHM_EXECUTION_ERROR = 462
```

Comment:
done

Reply:
I don't have a problem leaving this like that but there is no reason not to use the 500 status code if there is no additional information in the error message. The way it is now 512 == 500. There is no difference in the way it will be handled from the frontend. My idea of having a different status code was that it will also contain the node/datasets that are now unavailable, so the frontend could show that information instead of a generic 500 error code/message.

Reply:
what do you think about this latest solution?

Reply:
It would be ideal to include the datasets/node that went down but it's good enough for now. No problem from my side.